### PR TITLE
fix: add version in userAgent requestHeaders

### PR DESCRIPTION
**Pull request by jsimonweb**
Context:
Surface version in the User-Agent header. The change updates the userAgent value to include 'GoogleUtils.VERSION'.

### CODE REVIEW

**Comment by kolea2** [comment]
> GoogleUtils.VERSION might not work. It seems to be at 1.30.3 and should be at least 1.30.5, maybe 1.30.6.

Yes, it should be 1.30.5. That value should be auto incrementing with each release. I'll investigate that separately.

File: google-api-client/src/main/java/com/google/api/client/googleapis/services/AbstractGoogleClientRequest.java [hunk 1]
```diff
@@ -126,7 +126,7 @@ protected AbstractGoogleClientRequest(AbstractGoogleClient abstractGoogleClient,
     // application name
     String applicationName = abstractGoogleClient.getApplicationName();
     if (applicationName != null) {
-      requestHeaders.setUserAgent(applicationName + " " + USER_AGENT_SUFFIX);
+      requestHeaders.setUserAgent(applicationName + GoogleUtils.VERSION + " " + USER_AGENT_SUFFIX);
```
Comment (kolea2) [inline google-api-client/src/main/java/com/google/api/client/googleapis/services/AbstractGoogleClientRequest.java]: minor - do you want to leave a space or delimiter between application name and version number?

[the remark this answers is not in the document]
Comment (frankyn) [reply google-api-client/src/main/java/com/google/api/client/googleapis/services/AbstractGoogleClientRequest.java]: @jsimonweb could you confirm with a manual client to make sure this behaves as expected?

**Comment by chingor13** [comment]
#1419 addresses the VERSION constant skew. It should also fix the line length checkstyle.

**Comment by elharo** [comment]
I think this is mooted by #1425.

**Pull request closed without merging**